tests/endpoints/v1/test_broadcasts.py

```diff
@@ -10,7 +10,7 @@ class BroadcastsTest(unittest.TestCase):
     def test_search_params_underscores(self, client):
         client.broadcasts.search(
             broadcast_id='broadcast_id',
-            location__place_id='place_id',
+            location__origin='0,0', location__place_id='place_id',
```

Comment:
As we don't have a lot of tests here, maybe try it with a more plausible lat,lon to also validate that your regex handles the dot and the minus sign correctly?